test/wallets_v2_test.js

```diff
@@ -133,6 +133,32 @@ describe("POST /v2/wallets/show", function() {
   it("succeeds when the totpToken is disabled", function() {
     return this.submit({username:"mfa-disabled@stellar.org", walletId:new Buffer("mfa-disabled@stellar.org").toString("base64")}).expect(200);
   });
+
+  it("resets the totpKey and totpDisabled after the grace period has elapsed", function() {
+    var username = "mfa-disabled@stellar.org";
+    return this.submit({username:username, walletId:new Buffer(username).toString("base64")})
```

Comment:
The only thing I'd change is adding:
```js
.expect(200)
```
here and in the second test. The rest is OK! :+1: